test/SIL/Parser/basic.sil

```diff
@@ -754,6 +754,19 @@ bb0(%0 : $Builtin.Word, %1 : $Builtin.Word):
   return %3 : $()
 }
 
+// CHECK-LABEL: sil @test_realloc_tail_elems
+sil @test_tail_elems : $@convention(thin) (Builtin.Word, Builtin.Word) -> () {
+bb0(%0 : $Builtin.Word, %1 : $Builtin.Word):
```

Comment:
Can you add a serialization test as well? If you don't know where that is I can provide you with a quick link.

Reply:
I assume `test/SIL/Serialization/basic.sil` ?

How do you typically run these tests actually?